src/content/pt/2019/accessibility.md

```diff
@@ -0,0 +1,312 @@
+
+Para marcar cabeçalhos de tabela, basta usar a tag [`<th>`](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/th) (em vez de [`<td>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td)), ou qualquer um das regras ARIA [`columnheader`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role) ou [`rowheader`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role). Apenas 24,5% das páginas com tabelas marcam suas tabelas com um desses métodos. Portanto, os três quartos das páginas que optam por incluir tabelas sem cabeçalhos estão criando sérios desafios para os usuários de leitores de tela.
+
+Usar `<th>` e `<td>` era de longe o método mais comumente usado para marcar cabeçalhos de tabela. O uso dos papéis `columnheader` e `rowheader` era quase inexistente, com apenas 677 sites no total usando-os (0,058%).
+
+#### Legendas
+
+As legendas das tabelas por meio do elemento [`<caption>`](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/caption) são úteis para fornecer mais contexto para leitores de todos os tipos. Uma legenda pode preparar o leitor para receber as informações que sua mesa está compartilhando e pode ser especialmente útil para pessoas que podem se distrair ou ser interrompidas facilmente. Eles também são úteis para pessoas que podem perder seu lugar em uma mesa grande, como um usuário de leitor de tela ou alguém com deficiência intelectual ou de aprendizado. Quanto mais fácil você permitir que os leitores entendam o que estão analisando, melhor.
+
+Apesar disso, apenas 4,32% das páginas com tabelas apresentam legendas.
+
+## Compatibilidade com tecnologias assistivas
+
+### O uso de ARIA
+
+Uma das especificações mais populares e amplamente utilizadas para acessibilidade na web é o padrão [Accessible Rich Internet Applications](https://www.w3.org/WAI/standards-guidelines/aria/) (ARIA). Este padrão oferece uma grande variedade de atributos HTML adicionais para ajudar a transmitir o propósito por trás dos elementos visuais (ou seja, seu significado semântico) e de quais tipos de ações eles são capazes.
+
+Usar o ARIA de maneira correta e apropriada pode ser desafiador. Por exemplo, das páginas que utilizam atributos ARIA, encontramos 12,31% com valores inválidos aos seus atributos. Isso é problemático porque qualquer erro no uso de um atributo ARIA não tem efeito visual na página. Alguns desses erros podem ser detectados com o uso de uma ferramenta de validação automatizada, mas geralmente exigem o uso prático de software de assistência real (como um leitor de tela). Esta seção examinará como o ARIA é usado na web e, especificamente, quais partes do padrão são mais predominantes.
+
```

Comment:
Remove this extra line

```suggestion
```

Reply:
I have read the English version again and re-translated. @HakaCode Can you check if it sounds better? It is in my last commit.

Reply:
Think this comment is under the wrong bit of feedback.